extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -279,6 +284,27 @@ public void build(
                 }
             }
         }
+        // look for all implementations of abstract classes annotated @Path
+        do {
+            Set<DotName> tmpAbstract = new HashSet<>();
+            for (final DotName iface : pathAbstract) {
```

Comment:
Could we rename the variable? It's not an interface but an abstract class.

Reply:
No offense, but its often hard enough to get a PR approved with these kinds of style reviews.